crates/solver/src/settlement_simulation.rs

```diff
@@ -171,6 +171,7 @@ pub async fn simulate_before_after_access_list(
         from,
         input: transaction.input.0,
         to,
+        gas: Some(transaction.gas.as_u64()),
```

Comment:
Is it also required to supply the gas price here?
I assume [these](https://docs.tenderly.co/simulations-and-forks/how-to-simulate-a-transaction/transaction-parameters#general-transaction-parameters) are possible parameters?

Reply:
Gas price is irrelevant for both access list and gas used calculations.

Reply:
Technically, this may be important. See https://github.com/gnosis/gp-v2-services/issues/883

If its not too much work, I would also include the gas price here to prevent simulation issues ☝️

Reply:
Hm, interesting. The only thing I knew about is that we had some differences in transaction simulation when the sender does not have enough funds for transaction fee.
Anyway, for our piece of mind, let's add it: https://github.com/cowprotocol/services/issues/207